config.go

```diff
@@ -29,6 +29,35 @@ import (
 	"k8s.io/klog/v2"
 )
 
+type ChainValidationConfig struct {
+	// Path to the file containing root certificates that are acceptable to the
+	// log. The certs are served through get-roots endpoint.
+	RootsPemFile string
+	// If RejectExpired is true then the certificate validity period will be
+	// checked against the current time during the validation of submissions.
+	// This will cause expired certificates to be rejected.
+	RejectExpired bool
+	// If RejectUnexpired is true then CTFE rejects certificates that are either
+	// currently valid or not yet valid.
+	RejectUnexpired bool
```

Comment:
Not for this PR, but do we still need `RejectUnexpired`?
And if not, would the user setting `NotAfterStart` or `NotAfterLimit` to non-nil values be enough?

Reply:
Left a TODO. It's currently used by daedalus.